packages/installation-tests/installation-tests.sh

```diff
@@ -77,13 +79,35 @@ function test_playwright_global_installation {
     exit 1
   fi
   copy_test_scripts
-  if node sanity.js playwright chromium 2>/dev/null; then
-    echo "Should not be able to launch chromium without PLAYWRIGHT_BROWSERS_PATH variable!"
+  node sanity.js playwright none
+  PLAYWRIGHT_BROWSERS_PATH="${BROWSERS}" node sanity.js playwright
+}
+
+function test_playwright_global_installation_cross_package {
+  initialize_test "${FUNCNAME[0]}"
+
+  local BROWSERS="$(pwd -P)/browsers"
+  PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD=1 npm install ${PLAYWRIGHT_FIREFOX_TGZ}
+  PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD=1 npm install ${PLAYWRIGHT_WEBKIT_TGZ}
+  PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD=1 npm install ${PLAYWRIGHT_CHROMIUM_TGZ}
+  if [[ -d "${BROWSERS}" ]]; then
```

Comment:
Why would the directory ever exist here? Maybe you intended to pass `PLAYWRIGHT_BROWSERS_PATH="${BROWSERS}"` in the installation commands along with `PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD`?